packages/cli/src/parallel-tests/messages.types.ts

```diff
@@ -19,6 +19,7 @@ export interface InitMessage {
       padTime: boolean;
       shiftTime: boolean;
       networkStubbing: boolean;
+      accelerate: boolean | undefined;
```

Comment:
```suggestion
      accelerate: boolean;
```

Reply:
I thought about doing this, but did the undefined thing instead to avoid having the default value of 'false' copy and pasted into too many places (think we want to keep our default value for acceleration the same between all commands, otherwise it'd be confusing). So we pass a boolean or undefined through most the levels, and then there's a single place in replayEvents which includes the logic of setting to default value of false if undefined.

Reply:
Update: actually I think we can get best of both worlds — I’ll add a shared const with default values for common command parameters, move the default application to the command level, and use that.